pad_ui/PackDownloadPage.gd

```diff
@@ -0,0 +1,49 @@
+"""
+	Copyright 2020 Google LLC
+	
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+	
+		https://www.apache.org/licenses/LICENSE-2.0
+	
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+"""
+"""
+	Object used to render a list of PackDownloadDiv objects given an Array of packNames
+"""
+signal fetched_pack(pack_name)
+signal removed_pack(pack_name)
+
+extends Node2D
+
+const pack_download_div_scene = preload("res://pad_ui/pack_download_div.tscn")
```

Comment:
What does "div" stand for here? Like an html `<div>` or something else?

Reply:
Yeah I used this "div" naming here because this component has very similar purpose as that of html divisions.

I think maybe a better naming for this is to rename ```PackDownloadDiv``` to ```PackDownloadListItem```, and rename ```PackDownloadPage``` to ```PackDownloadList```.